src/addressfinder/reverseGeocode.tsx

```diff
@@ -0,0 +1,41 @@
+const addressFinderAPIKey = "ARFHPVK67QXM49BEWDL3";
+const reverseGeocodeURL = `https://api.addressfinder.io/api/nz/address/reverse_geocode/?key=${addressFinderAPIKey}&format=json`;
+
+export interface reverseGeocodeResp {
+  completions: addressResp[];
```

Comment:
As this is an interface, please capitalise to `export interface ReverseGeocodeResponse` and `AddressResponse` respectively.

Reply:
Addressed.